src/plugins/data_source_management/public/components/data_source_selectable/data_source_selectable.tsx

```diff
@@ -237,6 +238,7 @@ export class DataSourceSelectable extends React.Component<
           iconSide="left"
           size="s"
           disabled={this.props.disabled || false}
+          textProps={{ className: 'buttonTitle' }}
```

Comment:
what's the difference of text props vs className? Can we be more specific about the class name?

Reply:
So the test props is "Object of props passed to the <span/> wrapping the content's text/children only (not icon)" based on the oui document. Not much difference from className. Let me move to className.